app/scripts/controllers/deployment.js

```diff
@@ -21,11 +21,15 @@ angular.module('openshiftConsole')
                         OwnerReferencesService,
                         Logger,
                         ProjectsService,
-                        StorageService) {
+                        StorageService,
+                        LabelFilter) {
```

Comment:
We've started to alphabetize the dependencies when possible to try and keep our sanity :) The lists can get long.